resources/views/pages-legacy/download.blade.php

```diff
@@ -43,33 +43,59 @@
             <div>
                 <?php if ($emulator['download_url'] ?? false): ?>
                     <p class="embedded mb-2 text-right whitespace-nowrap">
-                        <?= Blade::render("<x-link href=\"" . $emulator['download_url'] . "\" >Download</x-link>") ?>
+                        <x-link
+                            href="{{ $emulator['download_url'] }}"
+                            target="_blank"
+                            class="plausible-event-name=Download+Link+Click plausible-event-emulator={{ $emulator['handle'] }}"
```

Comment:
Events are attached via classnames, see [here](https://plausible.io/docs/custom-event-goals) and [here](https://plausible.io/docs/custom-props/for-custom-events).